### PR TITLE
fix(county) remove admin middleware from route

### PR DESCRIPTION
**Pull request by NaamanTsur**
Currently the county display name route is defined as an admin route which it shouldn't be. This PR makes it a normal route.

### CODE REVIEW

**Review by ItayBenMoshe — DISMISSED** [review]

**Review by sandymash — APPROVED** [review]

File: server/src/middlewares/Authentication.ts [hunk 1]
```diff
@@ -50,25 +50,49 @@ const handleConfidentialAuth = (
...
-        return next();
+            step: `authorized azure token successfully! got user: ${JSON.stringify(user)}`
+        });
+
+        logger.info({
+            service: Service.SERVER,
+            severity: Severity.LOW,
+            workflow: 'Authentication',
+            step: `request to the graphql API with parameters: ${user.id}`,
+            user: response.locals.user.id,
```
Comment (RoeiRom) [inline server/src/middlewares/Authentication.ts:75]:
```suggestion
            user: user.id,
```
because the response.locals.user is not initated yet

**Review by ItayBenMoshe — APPROVED** [review]

**Review by MayaSasson — DISMISSED** [review]

File: server/src/middlewares/Authentication.ts [hunk 2]
```diff
@@ -50,25 +50,49 @@ const handleConfidentialAuth = (
...
+            investigation: +request.headers.epidemiologynumber
+        });
+
+        graphqlRequest(GET_USER_BY_ID, response.locals, { id: user.id }).then((result: any) => {
+            response.locals.user = {
```
Comment (RoeiRom) [inline server/src/middlewares/Authentication.ts:80]: add info log that got the response from the DB

**Review by eladyashar — APPROVED** [review]

File: server/src/middlewares/Authentication.ts (hunk 2 above)
Comment (sandymash) [inline server/src/middlewares/Authentication.ts:80]: whats the benefit of query the user on every request?
even if we dont need to get the isAdmin classification

File: server/src/middlewares/Authentication.ts [hunk 3]
```diff
@@ -109,17 +133,31 @@ const authMiddleware = (
...
                 service: Service.SERVER,
                 severity: Severity.LOW,
                 workflow: 'Authentication',
                 step: `noauth user found successfully, the user is: ${JSON.stringify(user)}`
             })
-            response.locals.user = user;
-            response.locals.epidemiologynumber = request.headers.epidemiologynumber;
-            return next();
+            graphqlRequest(GET_USER_BY_ID, response.locals, { id: user.id }).then((result: any) => {
+                response.locals.user = {
```
Comment (RoeiRom) [inline server/src/middlewares/Authentication.ts:145]: add info log

File: server/src/middlewares/Authentication.ts (hunk 2 above)
Comment (NaamanTsur) [reply server/src/middlewares/Authentication.ts:80]: We do need the investigation group to get the county

File: server/src/middlewares/Authentication.ts [hunk 4]
```diff
@@ -50,25 +50,49 @@ const handleConfidentialAuth = (
...
+            workflow: 'Authentication',
+            step: `request to the graphql API with parameters: ${user.id}`,
+            user: response.locals.user.id,
+            investigation: +request.headers.epidemiologynumber
+        });
+
+        graphqlRequest(GET_USER_BY_ID, response.locals, { id: user.id }).then((result: any) => {
+            response.locals.user = {
+                ...user,
+                ...result.data.userById
```
Comment (RoeiRom) [inline server/src/middlewares/Authentication.ts:82]: we dont need the whole user data, only the isAdmin and group

File: server/src/middlewares/Authentication.ts [hunk 5]
```diff
@@ -109,17 +133,31 @@ const authMiddleware = (
...
                 workflow: 'Authentication',
                 step: `noauth user found successfully, the user is: ${JSON.stringify(user)}`
             })
-            response.locals.user = user;
-            response.locals.epidemiologynumber = request.headers.epidemiologynumber;
-            return next();
+            graphqlRequest(GET_USER_BY_ID, response.locals, { id: user.id }).then((result: any) => {
+                response.locals.user = {
+                    ...user,
+                    ...result.data.userById
```
Comment (RoeiRom) [inline server/src/middlewares/Authentication.ts:147]: same like 82

**Review by RoeiRom — CHANGES_REQUESTED** [review]

File: server/src/middlewares/Authentication.ts (hunk 1 above)
Comment (NaamanTsur) [reply server/src/middlewares/Authentication.ts]: fixed

File: server/src/middlewares/Authentication.ts (hunk 2 above)
Comment (NaamanTsur) [reply server/src/middlewares/Authentication.ts:88]: fixed

File: server/src/middlewares/Authentication.ts (hunk 3 above)
Comment (NaamanTsur) [reply server/src/middlewares/Authentication.ts:171]: fixed

File: server/src/middlewares/Authentication.ts (hunk 4 above)
Comment (NaamanTsur) [reply server/src/middlewares/Authentication.ts]: fixed

File: server/src/middlewares/Authentication.ts (hunk 5 above)
Comment (NaamanTsur) [reply server/src/middlewares/Authentication.ts]: fixed

**Review by RoeiRom — APPROVED** [review]

**Review by eladyashar — APPROVED** [review]

**Pull request merged**